influxql/functions.gen.go.tmpl

```diff
@@ -4,12 +4,31 @@ package influxql
 
 // {{$k.Name}}PointAggregator aggregates points to produce a single point.
 type {{$k.Name}}PointAggregator interface {
-	Aggregate(p *{{.Name}}Point)
+	Aggregate(p *{{$k.Name}}Point)
+}
+
+// {{$k.Name}}BulkPointAggregator aggregates multiple points at a time.
+type {{$k.Name}}BulkPointAggregator interface {
```

Comment:
This just gives the capability for a bulk point aggregator to not implement `Aggregate()`. I don't know if there's a reason why that would be done, but it's possible. The reducer's that implement bulk should have both methods already. Do you need the bulk point aggregator interface to implement both?